db/migrations/49_rename_datetime_columns.sql

```diff
@@ -0,0 +1,876 @@
+    RENAME COLUMN `last_activity_date` TO `last_activity_at`,
+    RENAME COLUMN `thread_start_date` TO `thread_started_at`,
+    RENAME COLUMN `best_answer_date` TO `best_answer_at`,
+    RENAME COLUMN `last_answer_date` TO `last_answer_at`,
+    RENAME COLUMN `last_hint_date` TO `last_hint_at`,
+    RENAME COLUMN `contest_start_date` TO `contest_started_at`;
+ALTER TABLE `history_groups_groups`
+    RENAME COLUMN `status_date` TO `status_changed_at`;
+ALTER TABLE `history_groups_items`
+    RENAME COLUMN `partial_access_date` TO `partial_access_since`,
+    RENAME COLUMN `full_access_date` TO `full_access_since`,
+    RENAME COLUMN `access_solutions_date` TO `solutions_access_since`,
+    RENAME COLUMN `cached_full_access_date` TO `cached_full_access_since`,
+    RENAME COLUMN `cached_partial_access_date` TO `cached_partial_access_since`,
+    RENAME COLUMN `cached_access_solutions_date` TO `cached_solutions_access_since`,
+    RENAME COLUMN `cached_grayed_access_date` TO `cached_grayed_access_since`;
+ALTER TABLE `history_items`
+    RENAME COLUMN `access_open_date` TO `contest_opens_at`,
+    RENAME COLUMN `end_contest_date` TO `contest_closes_at`;
+ALTER TABLE `history_messages`
+    RENAME COLUMN `submission_date` TO `submitted_at`;
+ALTER TABLE `history_threads`
+    RENAME COLUMN `last_activity_date` TO `last_activity_at`;
+ALTER TABLE `history_users`
+    RENAME COLUMN `registration_date` TO `registered_at`,
+    RENAME COLUMN `last_login_date` TO `last_login_at`,
+    RENAME COLUMN `last_activity_date` TO `last_activity_at`,
+    RENAME COLUMN `notification_read_date` TO `notifications_read_at`;
+ALTER TABLE `history_users_items`
+    RENAME COLUMN `start_date` TO `started_at`,
+    RENAME COLUMN `validation_date` TO `validated_at`,
+    RENAME COLUMN `finish_date` TO `finished_at`,
+    RENAME COLUMN `last_activity_date` TO `last_activity_at`,
+    RENAME COLUMN `thread_start_date` TO `thread_started_at`,
+    RENAME COLUMN `best_answer_date` TO `best_answer_at`,
+    RENAME COLUMN `last_answer_date` TO `last_answer_at`,
+    RENAME COLUMN `last_hint_date` TO `last_hint_at`,
+    RENAME COLUMN `contest_start_date` TO `contest_started_at`;
+ALTER TABLE `history_users_threads`
+    RENAME COLUMN `last_read_date` TO `last_read_at`,
+    RENAME COLUMN `last_write_date` TO `last_wrote_at`;
+ALTER TABLE `items`
+    RENAME COLUMN `access_open_date` TO `contest_opens_at`,
+    RENAME COLUMN `end_contest_date` TO `contest_closes_at`;
+ALTER TABLE `login_states`
+    RENAME COLUMN `expiration_date` TO `expires_at`,
+    RENAME INDEX `expiration_date` TO `expires_at`;
+ALTER TABLE `messages`
+    RENAME COLUMN `submission_date` TO `submitted_at`;
+ALTER TABLE `sessions`
+    RENAME COLUMN `expiration_date` TO `expires_at`,
+    RENAME COLUMN `issued_at_date` TO `issued_at`,
+    RENAME INDEX `expiration_date` TO `expires_at`;
+ALTER TABLE `threads`
+    RENAME COLUMN `last_activity_date` TO `last_activity_at`;
```

Comment:
`latest_activity_at` would be more correct

The same apply also to many other "last_"" attributes  (will not repeat)

Reply:
fixed